gax/src/main/java/com/google/api/gax/batching/FlowController.java

```diff
@@ -155,30 +159,85 @@ public FlowController(FlowControlSettings settings) {
       case Ignore:
         this.maxOutstandingElementCount = null;
         this.maxOutstandingRequestBytes = null;
+        this.minOutstandingElementCount = null;
```

Comment:
Would it be possible to use a helper method to convert FlowControlSettings -> DynamicFlowControlSettings and have the old ctor just call the new ctor?

public FlowController(FlowControlSettings settings) {
   this(convertToDynamicSettings(settings));
}